### ISSUE TITLE
ClassNotFoundException: org.apache.logging.log4j.LogManager

### ISSUE THREAD

**Issue by adrien-cotte** [Merray/Splug#1]
Hi,

While compiling the projet, I get this error:

```
root@e1047241cf2e:/BotVouivrarium-master# mvn clean install
root@e1047241cf2e:/BotVouivrarium-master# java -jar ./target/botvouivrarium-1.0-SNAPSHOT.jar
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/logging/log4j/LogManager
        at fr.jim.mainapp.MainApp.<clinit>(MainApp.java:19)
Caused by: java.lang.ClassNotFoundException: org.apache.logging.log4j.LogManager
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:527)
        ... 1 more
```

Can you help me, please?

Best,
Adrien

**Comment by Merray** [comment]
Hello dear user, this issue has been fixed with commit 8463cbba4e6f02fc22e77b4ff6fd80e6a8a1ece9

Enjoy

**Comment by adrien-cotte** [comment]
Many thanks!
